verification/curator-service/ui/src/components/NewCaseForm.tsx

```diff
@@ -139,31 +148,48 @@ class NewCaseForm extends React.Component<Props, NewCaseFormState> {
                 {({ submitForm, isSubmitting, values }): JSX.Element => (
                     <div className={classes.container}>
                         <nav className={classes.tableOfContents}>
```

Comment:
Changes here LG. With the anticipated size of this form, would it be cleaner/easier to develop moving forward if we broke out both the ToC and a generalized form fieldset (/`Scroll.Element`) into separate components?

Reply:
Yes, probably. I believe the scrollTo(name) command works across files too. I'll submit this PR as is, and work on a refactoring one separately. I also want to double check I'm understanding the refactor the same way you are so will chat when you're online :)